test/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -1837,10 +1877,8 @@ parse_error_iq(#xmlel{name = <<"iq">>,
 generate_rpc_jid({_,User}) ->
     {username, Username} = lists:keyfind(username, 1, User),
     {server, Server} = lists:keyfind(server, 1, User),
-    %% esl-ejabberd uses different record to store jids
-     %JID = <<Username/binary, "@", Server/binary, "/rpc">>,
-     %{jid, JID, Username, Server, <<"rpc">>}.
-    {jid, Username, Server, <<"rpc">>, Username, Server, <<"rpc">>}.
+    JID = <<Username/binary, "@", Server/binary, "/rpc">>,
+    escalus_ejabberd:rpc(jlib, binary_to_jid, [JID]).
```

Comment:
There is simple function for lowering the jid in escalus: `escalus_utils:jid_to_lower/1` It doesn't do the full escaping and profile checks as `jlib:binary_to_jid/1` does, but it's good enough for tests.